### ISSUE TITLE
Binaries/builds

### ISSUE THREAD

**Issue by andybak** [Immersive-Foundation/Immersive_Foundation#3]
Hi, Do you plan to release binaries? I'm not a C++ dev and the project doesn't build out of the box on Windows. I don't have the expertise or inclination to fix the issues but I'm interested in working on the Unity side of things.

**Comment by ShacharWeis** [comment]
Same problem here. Were you able to build or locate the binaries?

**Comment by demonixis** [comment]
Hello, we cannot build the library without solid instructions. Can you provide binaries please?

**Comment by andybak** [comment]
Anyone?

**Comment by ShacharWeis** [comment]
I'm pretty sure this project is abandoned.

**Comment by stephen-radial** [comment]
You can extract the Unity binaries by switching to an earlier version of the project.  They still exist there and seem to work fine.

I would push some fixes that include these files, but I don't have write access to this project.

**Comment by andybak** [comment]
> You can extract the Unity binaries by switching to an earlier version of the project. They still exist there and seem to work fine.

Thanks. That got me a bit further but I'm getting "FAILED" when the dll initializes now.

> I would push some fixes that include these files, but I don't have write access to this project.

You could just fork it and push to your own copy?

**Comment by NickLadd** [comment]
Also hoping daily for the binaries to be pushed or for somebody to get this working.

**Comment by adrianmeyerart** [comment]
Yes, would be great. Hoping one of the devs can provide some more information how to get the build to work.

**Comment by stephen-radial** [comment]
> Thanks. That got me a bit further but I'm getting "FAILED" when the dll initializes now.

Make sure the antialiasing value you pass during initialization is 8.  It still ran for us when our actual program's AA value was lower, but the initialization method needs to be told it's 8 or it will fail.
